### PR TITLE
Remove deprecated headless option for chrome and firefox.

### PR DESCRIPTION
**Pull request by trauty-is-me**
Deprecated in Version 4.8.0 of Selenium
Removed in version 4.13.0

### CODE REVIEW

File: src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def create_headless_chrome(
     ):
         if not options:
             options = webdriver.ChromeOptions()
-        options.headless = True
+        options.add_argument('--headless')
```
Comment (emanlove) [inline src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py]: For Chrome we should be using `--headless=new`. [[Reference](https://www.selenium.dev/blog/2023/headless-is-going-away/)]

File: src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py [hunk 2]
```diff
@@ -244,7 +244,7 @@ def create_headless_firefox(
     ):
         if not options:
             options = webdriver.FirefoxOptions()
-        options.headless = True
+        options.add_argument('--headless')
```
Comment (emanlove) [inline src/SeleniumLibrary/keywords/webdrivertools/webdrivertools.py]: Checking (and double-checking this) but it looks like for Firefox it is `-headless`; that is with one hyphen("-") not two. [References: [1](https://www.selenium.dev/documentation/webdriver/browsers/firefox/#arguments), [2](https://hacks.mozilla.org/2017/12/using-headless-mode-in-firefox/), [3](https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/firefoxOptions)]

**Comment by emanlove** [comment]
@trauty-is-me Thanks for this pull request. I've made a couple of suggestions/corrections on the code. Let me know if you see anything I am missing.  I will schedule this for the next release v6.2.0. Thanks again and have a great day!

**Comment by trauty-is-me** [comment]
Happy to make the changes,

I won't be in a position to make the changes and test for a couple of days as I am away without my computer.

**Pull request merged**